packages/apps-config/src/ui/colors.ts

```diff
@@ -156,9 +156,9 @@ const nodeMathChain = '#000000';
 const nodeMoonbeam = '#53cbc9';
 const nodeMoonriver = '#0E132E';
 const nodeMybank = '#282736';
+const nodeNeumann = '#E38985';
```

Comment:
const nodeNeumann = '#E38985';

This needs to be changed to the new color too. Please review your own PR before submit it.